geodatasets/lib.py

```diff
@@ -118,6 +119,30 @@ def query_name(self, name: str) -> Dataset:
 
         raise ValueError(f"No matching item found for the query '{name}'.")
 
+    def filter_by_geometry(self, geometry_type: str) -> dict[str, Dataset]:
+        """Return dictionary of :class:`Dataset` objects filtered by geometry type
+
+        Returns dictionary of :class:`Dataset` objects from the :class:`Bunch`
+        filtered by `geometry_type` if found in the :list:`GEOMETRY_TYPES`
+
+        Parameters
+        ----------
+        geometry_type : str
+            GeometryType.name of the data item. Case does not matter.
```

Comment:
```suggestion
            Geometry type. The case does not matter.
```

Not sure what `GeometryType.name` means here. There's no such class.

Reply:
moved from enum to list[str] and forget to update docstring - my bad